python/tank/deploy/tank_commands/setup_project_core.py

```diff
@@ -277,44 +278,44 @@ def _project_setup_internal(log, sg, sg_app_store, sg_app_store_script_user, set
             project_name = setup_params.get_project_disk_name()
             log.debug("Shotgun: Setting Project.tank_name to %s" % project_name)
             sg.update("Project", project_id, {"tank_name": project_name})
-        
+
         log.info("Creating Pipeline Configuration in Shotgun...")
         data = {"project": sg_project_link,
                 "linux_path": config_location_linux,
                 "windows_path": config_location_win,
                 "mac_path": config_location_mac,
                 "code": constants.PRIMARY_PIPELINE_CONFIG_NAME}
-    
+
         # create pipeline configuration record
         pc_entity = sg.create(constants.PIPELINE_CONFIGURATION_ENTITY, data)
-        pipeline_config_id = pc_entity["id"] 
+        pipeline_config_id = pc_entity["id"]
         log.debug("Created data: %s" % pc_entity)
-    
+
     # write the record to disk
     pipe_config_sg_id_path = os.path.join(config_location_curr_os, "config", "core", "pipeline_configuration.yml")
```

Comment:
i wanted to make as few changes to the old setup code as possible, this is why this is not using the constant. Updated now!